src/distribution/gamma.rs

```diff
@@ -506,26 +460,22 @@ mod test {
         test_case(1.0, 1.0, 0.0, |x| x.min());
         test_case(10.0, 10.0, 0.0, |x| x.min());
         test_case(10.0, 1.0, 0.0, |x| x.min());
-        test_case(10.0, f64::INFINITY, 0.0, |x| x.min());
         test_case(1.0, 0.1, f64::INFINITY, |x| x.max());
         test_case(1.0, 1.0, f64::INFINITY, |x| x.max());
         test_case(10.0, 10.0, f64::INFINITY, |x| x.max());
         test_case(10.0, 1.0, f64::INFINITY, |x| x.max());
-        test_case(10.0, f64::INFINITY, f64::INFINITY, |x| x.max());
     }
 
     #[test]
     fn test_pdf() {
-        test_case(1.0, 0.1, 0.090483741803595961836995913651194571475319347018875963, |x| x.pdf(1.0));
-        test_case(1.0, 0.1, 0.036787944117144234201693506390001264039984687455876246, |x| x.pdf(10.0));
-        test_case(1.0, 1.0, 0.36787944117144232159552377016146086744581113103176804, |x| x.pdf(1.0));
-        test_case(1.0, 1.0, 0.000045399929762484851535591515560550610237918088866564953, |x| x.pdf(10.0));
+        test_almost(1.0, 0.1, 0.090483741803595961836995913651194571475319347018875963, 1e-10, |x| x.pdf(1.0));
```

Comment:
I'm curious as to why these changed and is `1e-10` the strictest bound on them?

Reply:
I can't explain why those are removed, must be an error on my part.

I'm not sure what a good bound is or how the values in the tests were obtained. Is there a rule to follow on this?

Reply:
Sorry for the confusion, I only saw the four lines outlined above and assumed I'd mistakenly removed tests.
In general testing exact values in floating point is a bad idea. I opened an issue #108 explaining my rationale.

Reply:
A lot of these were either ported directly from Math.NET or generated by comparing to other gamma distribution calculators if there were no comparable unit tests in the Math.NET library.

I definitely agree with #108 and as part of that issue it might be worth it to determine what a good bound should be. Anecdotally `1e-14` is probably as much precision as most people will need although there are definitely unit tests that don't meet that criteria (I know there's a couple that are only `1e-12`).

These are probably fine as is for now I was just asking out of curiousity